### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards-InputsWithValidation.json]: Luminosity ratio of selected time entry control with respect to its non-selected adjacent background color area is 1.67:1, which is less than minimum required ratio i.e., 3:1.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9225]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Tool: Color Contrast analyzer -> Launch through Accessibility Insights for Windows Tool

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'InputsWithValidation.json' control present on left navigation using tab key and activate it using enter key.
3. 'InputsWithValidation.json' adaptive card will appear on right sight of the pane.
4. Navigate to the 'Start time' combo box field using tab key and activate using enter key and select any time entry from it.
5. Launch Accessibility Insights for Windows tool and open Color contrast analyzer and select the foreground and its adjacent background color of the selected time entry with respect to its non-selected adjacent background.
6. Verify that, luminosity ratio of selected time entry control with respect to its non-selected adjacent background color area is less than minimum required ratio i.e., 3:1 or not.

**Actual Result:**
Luminosity ratio of selected time entry control with respect to its non-selected adjacent background color area is 1.67:1, which is less than minimum required ratio i.e., 3:1.

**Expected Result:**
Luminosity ratio of selected time entry control with respect to its non-selected adjacent background color area should be greater than or equal to the minimum required ratio i.e., 3:1.

**User Impact:**
Users with visual impairments, especially those with low vision or color blindness, may struggle to distinguish between the selected and non-selected controls due to the insufficient contrast due to which users might not distinguish the selected and non-selected control.

**MAS Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/51036e08-49f8-4cc1-99f5-c92fe9774631)

### Card JSON

```JSON
N/A
```
